### PR TITLE
Rich Emoji Support

### PR DESCRIPTION
**Pull request by atomhax**
Added rich emoji support. It was kinda a hackish solution to force the react core to update. Comments and critiques are always welcome.

### CODE REVIEW

**Comment by atomhax** [comment]
I only tested this via macOS terminal. I'm requesting help from anyone willing to test this feature as well.

**Comment by timgrossmann** [comment]
@atomhax Awesome, thank you!

Could you please remove the .DS_Store File?

File: README.md [hunk 1]
```diff
@@ -137,13 +137,23 @@ session.set_comments(['Nice shot!'], media='Photo')
 session.set_comments(['Great Video!'], media='Video')
 ```
 
-> **Emoji Support**  
+##### Emoji Support  
+
+You can use Unicode characters (like Emoji) in your comments
+1. You have to convert your comment to Unicode. This can safely be done by adding an u in front of the opening apostrophe:
```
Comment (timgrossmann) [inline README.md:143]: Could you please also add this to the docs/index.md file?

**Review by timgrossmann — CHANGES_REQUESTED** [review]

**Comment by atomhax** [comment]
@timgrossmann oops. I removed the .DS_Store files from the PR.

File: README.md (hunk 1 above)
Comment (atomhax) [reply README.md:143]: all set

**Review by timgrossmann — APPROVED** [review]

**Pull request merged**